tests/mockupDrivers/MockupVideoDriver.cpp

```diff
@@ -81,3 +81,9 @@ void* MockupVideoDriver::GetMapPointer() const
 {
     return nullptr;
 }
+
+void MockupVideoDriver::ShowErrorMessage(const char* title, const char* message)
+{
+    std::cout << title;
```

Comment:
`#include <iostream>` is missing in this file, also probably some newlines after the title/message should be good?